packages/scripts/src/commands/app-data/convert/processors/flowParser/parsers/template.parser.ts

```diff
@@ -58,14 +56,19 @@ export class TemplateParser extends DefaultParser {
 
```

Comment:
The lines in the final if statement above throws some TypeScript errors. These are present on master too so not necessary to fix, but this could be an opportunity to do so (e.g. by asserting `as FlowTypes.IDynamicField`?)

<img width="800" alt="Screenshot 2024-04-08 at 12 12 51" src="https://github.com/IDEMSInternational/parenting-app-ui/assets/64838927/04c89114-7676-4212-a346-6513f443710a">

Reply:
I think asserting the type kinda overlooks the issue that the function isn't written very well given the ambiguous return depending on type of data input. 

Given that trying to fix this here might have knock-ons for the outputs we're trying to review against I think better this handled in a follow-up PR instead. I'll try to draft something that can be merged after this and once debug content repo updated

Reply:
Will follow-up in #2294